Extensions/TweenBehavior/tweenruntimebehavior.ts

```diff
@@ -1,7 +1,50 @@
+/// <reference path="shifty.d.ts" />
+const logger = new gdjs.Logger('Tween behavior');
```

Comment:
If I'm not mistaken, this will create the logger in the global namespace (i.e: global variable), you should put it inside `namespace gdjs {`

Reply:
Whoops 😬